src/barnetilsyn/steg/6-barnepass/Barnepass.tsx

```diff
@@ -87,9 +89,50 @@ const Barnepass: FC<Props> = () => {
         },
       };
     });
-    settDokumentasjonsbehov(spørsmål, svar);
   };
 
+  useEffect(() => {
+    const søkerDuStønadFraBestemtMånedSvar:
+      | ISvar
+      | undefined = SøkerDuStønadFraBestemtMndSpm.svaralternativer.find(
+      (alternativ) => alternativ.id === søkerFraBestemtMåned?.svarid
+    );
+    const skalLeggeVedTidligereFakturaer = skalLeggeVedDokumentasjonPåTidligereFakturaer(
+      barnSomSkalHaBarnepass,
+      søkerFraBestemtMåned,
+      søknadsdato
+    );
+
+    if (søkerDuStønadFraBestemtMånedSvar !== undefined) {
+      const dokumentasjonsbehovFinnesFraFør = søknad.dokumentasjonsbehov.some(
+        (dok) => dok.id === BarnetilsynDokumentasjon.TIDLIGERE_FAKTURAER
+      );
+      if (dokumentasjonsbehovFinnesFraFør && !skalLeggeVedTidligereFakturaer) {
+        settDokumentasjonsbehov(SøkerDuStønadFraBestemtMndSpm, {
+          ...søkerDuStønadFraBestemtMånedSvar,
+          dokumentasjonsbehov: undefined,
+        });
+      } else if (
+        !dokumentasjonsbehovFinnesFraFør &&
+        skalLeggeVedTidligereFakturaer
+      ) {
+        settDokumentasjonsbehov(
+          SøkerDuStønadFraBestemtMndSpm,
+          søkerDuStønadFraBestemtMånedSvar
+        );
+      }
+    }
+    // eslint-disable-next-line
+  }, [søknadsdato, søkerFraBestemtMåned, barnSomSkalHaBarnepass]);
```

Comment:
Basert på https://favro.com/organization/98c34fb974ce445eac854de0/a64c6aad9b0d61ef6c0290bd?card=Tea-1876
tror jeg vi kan slette alt som har med dokumentasjonsbehovet for "tidligere fakturaer" å gjøre - og kun vise teksten... Da blir koden et par knepp ryddigere!